development/nautobot_config.py

```diff
@@ -19,8 +24,24 @@
     }
 }
 
+HIDE_RESTRICTED_UI = os.getenv("HIDE_RESTRICTED_UI", False)
+
+SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "")
```

Comment:
Why move these up here? They don't seem to belong under the `Databases` grouping of configs?